NYUDining/EditProfileViewController.swift

```diff
@@ -0,0 +1,143 @@
+        case SPS = 11
+        case SL = 12
+        case SM = 13
+        case SSSW = 14
+        case SSCEH = 15
+        case TSE = 16
+        case TSA = 17
+        case AD = 18
+        case SH = 19
+        
+        static var count: Int { return NYUSchools.SH.rawValue + 1 }
+        
+        var description: String {
+            switch self {
+            
+            case .CAS : return "College of Arts & Science"
+            case .GSAS : return "Graduate School of Arts and Science"
+            case .CD : return "College of Dentistry"
+            case .CGPH : return "College of Global Public Health"
+            case .CI : return "Courant Institute"
+            case .GS : return "Gallatin School"
+            case .IFA : return "Institute of Fine Arts"
+            case .ISAW : return "Institute for the Study of the Ancient World"
+            case .SSB : return "Leonard N. Stern School of Business"
+            case .GSPS : return "Robert F. Wagner Graduate School of Public Service"
+            case .CN : return "Rory Meyers College of Nursing"
+            case .SPS : return "School of Professional Studies"
+            case .SL : return "School of Law"
+            case .SM : return "School of Medicine"
+            case .SSSW : return "Silver School of Social Work"
+            case .SSCEH : return "Steinhardt School of Culture, Education, and Human Development"
+            case .TSE : return "Tandon School of Engineering"
+            case .TSA : return "Tisch School of the Arts"
+            case .AD : return "NYU Abu Dhabi"
+            case .SH : return "NYU Shanghai"
+            default : return " "
+            }
+        }
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    func numberOfComponentsInPickerView(pickerView: UIPickerView) -> Int {
+        return 1
+    }
+    
+    func pickerView(pickerView: UIPickerView, numberOfRowsInComponent component: Int) -> Int {
+        if (pickerView == schoolPicker)
+        {
+            return 20
+        }
+        return 1
+    }
+    
+    func pickerView(pickerView: UIPickerView, titleForRow row: Int, forComponent component: Int) -> String! {
+        if (pickerView == schoolPicker)
+        {
+            let Nschools = NYUSchools(rawValue: (row))
+            return Nschools?.description
+        }
+        return "nothing"
+    }
+    func pickerView(pickerView: UIPickerView, didSelectRow row: Int, inComponent component: Int) {
+        let Nschools = NYUSchools(rawValue: row)
+        
+        print("Nschools.description: \(Nschools!.description)")
+        
+        let defaults = NSUserDefaults.standardUserDefaults()
+        defaults.setObject(Nschools?.description, forKey: "possibleSchool")
+
+    }
+    
+    @IBAction func saveInfo(sender: AnyObject) {
+        print ("save button is pressed")
+        let defaults = NSUserDefaults.standardUserDefaults()
+        defaults.setObject(year.text, forKey: "School_Year")
```

Comment:
Not a good usage of NSUserDefaults. It would be better to have a flow like this: on initial login, collect user's information (school, major, etc.) and create a new user on the backend with that information. When going to display the information, the app should call the backend to retrieve all of the information. There should be a new User model class/struct that stores this information. Using NSUserDefaults can cause a ton of bugs, especially if it's not cleared properly on logout.